pkgs/development/libraries/utf8proc/default.nix

```diff
@@ -0,0 +1,25 @@
+{ fetchurl, stdenv }:
+
+stdenv.mkDerivation rec {
+  version = "v1.1.6";
+
+  name = "utf8proc-${version}";
+
+  src = fetchurl {
+    url = "http://www.public-software-group.org/pub/projects/utf8proc/${version}/utf8proc-${version}.tar.gz";
+    sha256 = "1rwr84pw92ajjlbcxq0da7yxgg3ijngmrj7vhh2qzsr2h2kqzp7y";
+  };
+
+  installPhase = ''
+    mkdir -pv $out/lib $out/include
+    cp libutf8proc.so libutf8proc.a $out/lib
+    cp utf8proc.h $out/include
+  '';
+
+  meta = {
+    description = "utf8proc is a library for processing UTF-8 encoded Unicode strings";
```

Comment:
Please do not repeat the package name in its short description (see http://nixos.org/nixpkgs/manual/#idm140737319319584).

Looks good otherwise.